hardhat.config.ts

```diff
@@ -4,8 +4,17 @@ import '@typechain/hardhat'
 import "@nomiclabs/hardhat-ethers";
 import "@nomiclabs/hardhat-waffle";
 
+require('dotenv').config()
+
 const config: HardhatUserConfig = {
-  defaultNetwork: "hardhat",
+  networks: {
+    hardhat: {
+      forking: {
+        url: process.env.FORK as string,
```

Comment:
Style nit: Can you pass in the env var similarly to how we did it in `governance-private`:

`      url: `https://eth-mainnet.alchemyapi.io/v2/${ALCHEMY_KEY}`,`